rllib/agents/a3c/README.md

```diff
@@ -0,0 +1,22 @@
+# Advantage Actor-Critic (A2C, A3C)
+
+## Overview 
+
+[Advantage Actor-Critic](https://arxiv.org/pdf/1602.01783.pdf) proposes two distributed model-free off-policy RL algorithms, A3C and A2C. These algorithms are distributed versions of DQN with different distributed execution patterns. The paper accelerates training via scaling data collection, i.e. introducing worker nodes, which carry copies of the central node's Q-network, that collect data from the environment in parallel. 
+
+In A2C, the worker nodes synchronously collect data. The collected data forms a giant batch of data, from which the central node (the central policy) computes gradient updates. On the other hand, in A3C, the worker nodes generate data asynchronously, compute gradients from the data, and send computed gradients to the central node. Note that the workers in A3C may be slightly out-of-sync with the central node due to asynchrony, which may induce biases in learning.
```

Comment:
Also mention here that the central node applies each of these gradients and then sends updated weights back to the workers.